deployment/base/topologyupdater-daemonset/topologyupdater-daemonset.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nfd
+  name: nfd-topology-updater
+spec:
+  selector:
+    matchLabels:
+      app: nfd-topology-updater
+  template:
+    metadata:
+      labels:
+        app: nfd-topology-updater
+    spec:
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: nfd-topology-updater
+          image: gcr.io/k8s-staging-nfd/node-feature-discovery:master
+          imagePullPolicy: Always
+          command:
+            - "nfd-topology-updater"
+          args:
+            - "--kubelet-config-file=/podresources/config.yaml"
+            - "--podresources-socket=/podresources/kubelet.sock"
+            - "--sleep-interval=3s"
+            - "--watch-namespace=rte"
```

Comment:
Well, I am not too bothered either way but I feel having the flags specified explicitly (or in this case the ones that would most likely have to be updated) in the pod spec can provide more visibility on how the daemon was configured. In case the default values of the flags are not suitable for the cluster (e.g. kubeletconfig at a different location), just inspecting the pod spec, the admin would have more visibility into the flags with which nfd-topology-updater was run rather than having to refer back to the code to determine the default arguments. Let me know what you would like and I will make changes if needed accordingly.

Reply:
Fair enough. Makes sense and I'm ok with this. However, I think that the values here should  be same as the defaults.

Reply:
Sure, done!